hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/utils/SynthTxnUtilsTest.java

```diff
@@ -44,6 +43,9 @@
 import org.junit.jupiter.api.Test;
 
 class SynthTxnUtilsTest {
+
+    private static final String LAZY_CREATION_MEMO = "";
```

Comment:
nit: similar to Thomas wondering if it's worth keeping this around.
If there needs to be a static empty memo field I'd imagine it already exists and would be applicable across services.
For instance do we check for empty token memo in this way and if so why not use that field, or do we do a memo.isEmpty().
Non blocking food for thought for a future refactor.

Reply:
Good question. I don't think we have an empty memo field, because this is the default if you do not set anything. I think most tests do not check default values of newly created objects. Definitely makes sense to check and fix.